edison/src/test/java/edu/illinois/cs/cogcomp/edison/utilities/CreateTestTAResource.java

```diff
@@ -46,7 +46,7 @@ public CreateTestTAResource() {
             // since the sentence below are not properly tokenized
             Map<String, String> nonDefaultValues = new HashMap<>();
             nonDefaultValues.put(CuratorConfigurator.RESPECT_TOKENIZATION.key,
-                    CuratorConfigurator.TRUE);
+                    CuratorConfigurator.FALSE);
```

Comment:
I don't think so: the problem arose in the TextAnnotation method that converts from Curator data structures, so the tests in the Curator module are the ones that are relevant.